### PR TITLE
Add: Dorksgek config file

### PR DESCRIPTION
**Pull request by LordAro**

### CODE REVIEW

**Comment by TrueBrain** [comment]
Please never copy the OpenTTD one as an example; it is a special case example :)

Use examples like:
https://github.com/OpenTTD/bananas-api/commit/bd7013d7ec527fb07f965d23a36b3be6b3ac6fda

Pretty please :)

**Comment by LordAro** [comment]
I copied it from NML :p

But I did wonder...

File: .dorpsgek.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+notifications:
+  global:
+    irc:
+      - openttd
+      - openttd.notice
+
+  pull-request:
+  issue:
+  tag-created:
+
```
Comment (TrueBrain) [inline .dorpsgek.yml:10]: E_TOO_MANY_NEWLINES

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,19 @@
+objs/
```
Comment (TrueBrain) [inline .gitignore:1]: why do some earn a /, and others do not?

**Review by TrueBrain — APPROVED** [review]

File: .gitignore (hunk 2 above)
Comment (LordAro) [reply .gitignore:1]: Quick and dirty conversion from the hgignore. No particular reason :)

**Review by TrueBrain — APPROVED** [review]

**Pull request merged**